themes/default/layouts/partials/head.html

```diff
@@ -153,7 +153,7 @@
         {{ else if eq .Type "docs" }}
             <title>{{ $title }} | Pulumi Docs</title>
         {{ else if eq .Type "registry" }}
-            <title>{{ $title }} | Pulumi Registry</title>
+            <title>{{ replace $title "/" "." }} | Pulumi Registry</title>
```

Comment:
google is cool with dots?

Reply:
i'm guessing it is since the resources in the other packages have dots in the title.